testhelpers/org.eclipse.n4js.ui.tests.helper/src/org/eclipse/n4js/tests/hover/AbstractN4JSHoverTest.xtend

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2019 HAW Hamburg.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Philip Aguilar Bremer, Max Neuwirt; HAW Hamburg
+ */
+package org.eclipse.n4js.tests.hover
+
+import org.eclipse.xtext.ui.testing.AbstractHoverTest
+import org.eclipse.jface.text.Region
+
+/**
+ * An abstract class that can return the html text of a hover to test if it matches a particular regex.
```

Comment:
Maybe add a line as follows:

```
Concrete test subclassing this class needs to have a suffix "PluginUITest" in order to be correctly executed by Maven.
```

in order to avoid problems when we re-use this class again.